static/app/stores/savedSearchesStore.tsx

```diff
@@ -66,7 +66,6 @@ const storeConfig: SavedSearchesStoreDefinition = {
           !(
             savedSearch.isPinned &&
             savedSearch.type === type &&
-            !savedSearch.isOrgCustom &&
```

Comment:
`isPinned` already implies it's not org custom (since all other searches are "org custom"